docker/ycsb/Dockerfile

```diff
@@ -16,6 +16,9 @@ RUN cd /YCSB/scylla/target && mkdir -p YCSB && tar xvvf ycsb-scylla-*.tar.gz -C
 
 
 FROM openjdk:8 as app
+
+RUN apt-get update && apt-get install -y python3
```

Comment:
It will make the final image be bigger than it can be if we add 'python3' installation to the lines 3-5

Reply:
we need that in run time, not only in build time.

Reply:
`RUN` command is used only in the build stage and the probability that some new update appears after lines 3-5 is about `0`. Also, `python3` version won't be updated that fast too. So, I don't see any relation of this new line to the statement about the `runtime` and my original concern is still valid.

Reply:
this is a multi stage build, put the python3 install on the first stage isn't going to help, it won't be available on the 2nd stage, nor in the final image.

I don't understand what are you commenting about

Reply:
Oh, I mislooked that we are in the different layer. No concern then.